pkg/runs/runsGet_test.go

```diff
@@ -1204,10 +1226,100 @@ func TestRunsGetURLQueryWithInvalidResultSuppliedReturnsError(t *testing.T) {
 	mockTimeService := utils.NewMockTimeService()
 
 	// When...
-	err := GetRuns(runName, age, requestor, result, outputFormat, mockTimeService, mockConsole, apiServerUrl)
+	err := GetRuns(runName, age, requestor, result, shouldGetActive, outputFormat, mockTimeService, mockConsole, apiServerUrl)
 
 	// Then ...
 	assert.Error(t, err)
 	assert.ErrorContains(t, err, "GAL1087E")
 	assert.ErrorContains(t, err, result)
 }
+
+func TestActiveAndResultAreMutuallyExclusiveShouldReturnError(t *testing.T) {
+	// Given ...
+	age := ""
+	runName := "U456"
+	requestor := ""
+	result := "Passed"
+	shouldGetActive := true
+
+	server := NewRunsGetServletMock(t, http.StatusOK, runName, RUN_U456)
+	defer server.Close()
+
+	outputFormat := "summary"
+	mockConsole := utils.NewMockConsole()
+
+	apiServerUrl := server.URL
+	mockTimeService := utils.NewMockTimeService()
+
+	// When...
+	err := GetRuns(runName, age, requestor, result, shouldGetActive, outputFormat, mockTimeService, mockConsole, apiServerUrl)
+
+	// Then ...
+	assert.Error(t, err)
+	assert.ErrorContains(t, err, "GAL1088E")
+}
+
+func TestActiveParameterReturnsOk(t *testing.T) {
+	// Given ...
+	age := ""
+	runName := "U456"
+	requestor := ""
+	result := ""
+	shouldGetActive := true
+
+	server := NewRunsGetServletMock(t, http.StatusOK, runName, RUN_U456)
+	defer server.Close()
+
+	outputFormat := "summary"
+	mockConsole := utils.NewMockConsole()
+
+	apiServerUrl := server.URL
+	mockTimeService := utils.NewMockTimeService()
+
+	// When...
+	err := GetRuns(runName, age, requestor, result, shouldGetActive, outputFormat, mockTimeService, mockConsole, apiServerUrl)
+
+	// Then ...
+	assert.Nil(t, err)
+}
+
+// func TestRunsGetActiveRunsBuildsQueryCorrectly(t *testing.T) {
+// 	// Given ...
+// 	age := ""
+// 	runName := "U456"
+// 	requestor := ""
+// 	result := ""
+// 	shouldGetActive := true
+
+// 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+// 		query := r.URL.Query()
+// 		assert.EqualValues(t, query.Get("from"), "")
+// 		assert.EqualValues(t, query.Get("to"), "")
+// 		assert.EqualValues(t, query.Get("runname"), runName)
+// 		assert.EqualValues(t, query.Get("requestor"), requestor)
+// 		assert.NotContains(t, r.URL.RawQuery, "status="+url.QueryEscape("finished"))
+// 		w.Header().Set("Content-Type", "application/json")
+// 		w.WriteHeader(200)
+// 		w.Write([]byte(`
+// 		 {
+// 			 "pageNumber": 1,
+// 			 "pageSize": 1,
+// 			 "numPages": 1,
+// 			 "amountOfRuns": 0,
+// 			 "runs":[]
+// 		 }`))
+// 	}))
+// 	defer server.Close()
+
+// 	outputFormat := "summary"
+// 	mockConsole := utils.NewMockConsole()
+
+// 	apiServerUrl := server.URL
+// 	mockTimeService := utils.NewMockTimeService()
+
+// 	// When...
+// 	err := GetRuns(runName, age, requestor, result, shouldGetActive, outputFormat, mockTimeService, mockConsole, apiServerUrl)
+
+// 	// Then ...
+// 	assert.Nil(t, err)
+// }
```

Comment:
As above, why is this commented out? If for a reason then please can you add a comment above describing why it's commented, so if someone else comes across it they know. If it's not needed then please remove.

Reply:
refactored to check the status is passed correctly in the url query